### PR TITLE
Update docs to reflect current builds of Suricata

### PR DESCRIPTION
**Pull request by utkonos**
I'm not sure of the other packages, but the PPA on Launchpad has bundled suricata-update since 4.1.x.
https://launchpad.net/~oisf/+archive/ubuntu/suricata-stable

Make sure these boxes are signed before submitting your Pull Request
-- thank you.

- [ ] I have read the contributing guide lines at
  https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [ ] I have signed the Open Information Security Foundation
  contribution agreement at
  https://suricata-ids.org/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the
  changes made (if applicable)

Link to [redmine](https://redmine.openinfosecfoundation.org/projects/suricata/issues) ticket:

Describe changes:
-
-
-

### CODE REVIEW

File: doc/quickstart.rst [hunk 1]
```diff
@@ -16,8 +16,9 @@ Pip can install ``suricata-update`` globally making it available to
 all users or it can install ``suricata-update`` into your home
 directory for use by your user.
 
-.. note:: At some point ``suricata-update`` should be bundled with
-          Suricata avoid the need for a separate installation.
+.. note:: Version 4.1+ from the Ubuntu PPA is bundled with
```
Comment (inashivb) [inline doc/quickstart.rst:19]: Hi @utkonos !
Thanks for your first contribution to the project. :)
I was wondering if we should drop the "from Ubuntu PPA" part as it was bundled for every distribution. https://suricata-ids.org/2018/11/06/suricata-4-1-released/

Comment (utkonos) [reply doc/quickstart.rst:19]: > Hi @utkonos !
> Thanks for your first contribution to the project. :)
> I was wondering if we should drop the "from Ubuntu PPA" part as it was bundled for every distribution. https://suricata-ids.org/2018/11/06/suricata-4-1-released/

That's great. Even better.

**Comment by jasonish** [comment]
Ping @utkonos ?

**Comment by inashivb** [comment]
Hi @utkonos !
We'll be happy to include these changes in the next major release. Could you please sign the contribution agreement?
Thanks for your work!

**Comment by jasonish** [comment]
Closing due to inactivity from the author. A note about it being included in 4.1+ exists in the master branch already.

**Pull request closed without merging**